process/smartContract/process_test.go

```diff
@@ -855,13 +851,13 @@ func TestScProcessor_ExecuteBuiltInFunctionSCRTooBig(t *testing.T) {
 		return acntSrc, nil
 	}
 
-	sc.flagSCRSizeInvariantOnBuiltInResult.SetValue(false)
 	retCode, err := sc.ExecuteBuiltInFunction(tx, acntSrc, nil)
 	require.Equal(t, vmcommon.Ok, retCode)
 	require.Nil(t, err)
 
 	_ = acntSrc.AddToBalance(big.NewInt(100))
-	sc.flagSCRSizeInvariantOnBuiltInResult.SetValue(true)
+	enableEpochsHandlerStub.IsSCRSizeInvariantOnBuiltInResultFlagEnabledField = true
+	enableEpochsHandlerStub.IsSCRSizeInvariantCheckFlagEnabledField = true
```

Comment:
why do we need to set this flag also?

Reply:
the flag is needed not to bypass [this check](https://github.com/ElrondNetwork/elrond-go/blob/integrate_enable_epochs_handler_p2/process/smartContract/process.go#L964).. the scope of the test is `SCRTooBig`

Reply:
ok